examples/speech_recognition/SpeechRecognitionExample.js

```diff
@@ -0,0 +1,224 @@
+      let arkScore = await this._getTensorArrayByArk(this._currentModelInfo.scoreFile)
+
+      this._initError(totalError);
+      for (let i = 0; i < arkInput.rows; i++) {
+        inputTensor.set(arkInput.data.subarray(i * arkInput.columns, (i + 1) * arkInput.columns));
+        scoreTensor.set(arkScore.data.subarray(i * arkScore.columns, (i + 1) * arkScore.columns));
+        status = await this._runner.run(inputTensor);
+        let output = this._runner.getOutput();
+        totalTime += output.inferenceTime;
+        this._compareScores(output.outputTensor, scoreTensor, 1, arkScore.columns, frameError);
+        this._updateScoreError(frameError, totalError);
+      }
+      this._result.errors = this._getReferenceCompareResults(totalError, arkScore.rows);
+      this._result.cycles = arkScore.rows;
+      this._result.time = totalTime.toFixed(2);
+
+      console.log(`Computed Status: [${status}]`);
+      console.log(`Compute Time: [${totalTime} ms]`);
+      this._processOutput();
+    } catch (e) {
+      showAlertComponent(e);
+      showErrorComponent();
+    }
+  }
+
+  // The function refer with speech_sample https://github.com/opencv/dldt/blob/2020/inference-engine/samples/speech_sample/main.cpp#L166.
+  _scaleFactorForQuantization = (data, targetMax, numElements) => {
+    let max = 0.0;
+    let scaleFactor;
+
+    for (let i = 0; i < numElements; i++) {
+      let absData = Math.abs(data[i]);
+      if (absData > max) {
+        max = absData;
+      }
+    }
+    if (max == 0) {
+      scaleFactor = 1.0;
+    } else {
+      scaleFactor = targetMax / max;
+    }
+
+    return scaleFactor;
+  }
+
+  _getTensorArrayByArk = async (arkPath) => {
+    let request = new Request(arkPath);
+    let response = await fetch(request);
+    let arkArrayBuffer = await response.arrayBuffer();
+    let arkBytesArray = new Uint8Array(arkArrayBuffer);
+    let EOF = arkBytesArray.findIndex((value) => (value == 4));  // find control-D (EOF)
+    let rowsBuffer = new Uint8Array(arkBytesArray.subarray(EOF+1, EOF+5)).buffer;     // read buffer of rows
+    let columnsBuffer = new Uint8Array(arkBytesArray.subarray(EOF+6, EOF+10)).buffer; // read buffer of columns
+    let arkRows = new Int32Array(rowsBuffer)[0];        // read number of rows
+    let arkColumns = new Int32Array(columnsBuffer)[0];  // read number of columns
+    let dataLength = arkRows * arkColumns * 4;
+    let dataBuffer = new Uint8Array(arkBytesArray.subarray(EOF+10, EOF+10+dataLength)).buffer;  // read buffer of data
+    let arkData = new Float32Array(dataBuffer);  // read number of data
+    let inputScaleFactor = this._scaleFactorForQuantization(arkData, this.targetMax, arkRows * arkColumns);
+
+    return {
+      rows: arkRows,
+      columns: arkColumns,
+      data: arkData,
+      inputScaleFactor: inputScaleFactor,
+    }
+  }
+
+  _initError = (error) => {
+    error.numScores = 0,
+    error.numErrors = 0,
+    error.threshold = 0.0001,
```

Comment:
@miaobin @mingmingtasd @fujunwei Is there some reference for this number **0.0001** ?

Reply:
Refferring to the definition of `#define MAX_SCORE_DIFFERENCE 0.0001f` in native ASR sample, we use the same value of 0.0001 as the limitation to compare with the max error value. We may need to update this value according to different backends (polyfill/GNA_HW) in the future.

Reply:
ditto. Please add references to where they come from. Thanks.